src/radio/index.tsx

```diff
@@ -42,15 +41,25 @@ export interface RadioProperties extends ThemeProperties, FocusProperties {
 	widgetId?: string;
 }
 
-const factory = create({ focus, theme }).properties<RadioProperties>();
+export interface RadioChild {
+	label?: () => RenderResult;
```

Comment:
this should just be a single child of type `RenderResult`, ie. a normal child. No need for the function or the object as nothing is injected.